google-cloud/terraform/terraform.tfvars

```diff
@@ -0,0 +1,42 @@
+# Google Cloud Project
+# project_id = "project-id"
+
+# Google Cloud Service Account Name
+service_account_name = "xtdb-service-account"
+
+# (Planned) Kubernetes namespace/service account
+# Used for the GKE Workload Identity with the Service Account
+kubernetes_namespace          = "xtdb-deployment"
+kubernetes_service_account_name = "xtdb-service-account"
+
+# Google Cloud Storage Settings
+storage_bucket_name          = "xtdb-storage-bucket"
+storage_bucket_location      = "us-central1"
+storage_bucket_storage_class = "STANDARD"
+
+# VPC/Network naming for XTDB VPC
+vpc_name          = "xtdb-vpc"
+vpc_subnet_name   = "xtdb-vpc-public-subnet"
+vpc_subnet_ip     = "10.10.10.0/24"
+vpc_subnet_region = "us-central1"
+
+# Key GKE Config
+kubernetes_cluster_name   = "xtdb-cluster"
+kubernetes_cluster_region = "us-central1"
+kubernetes_cluster_zones  = ["us-central1-a"]
+
+## GKE System Node Pool Settings
+system_node_pool_machine_type = "n2-standard-2"
+system_node_pool_locations    = "us-central1-a"
+system_node_pool_min_count    = 1
+system_node_pool_max_count    = 1
+system_node_pool_disk_size_gb = 50
+system_node_pool_disk_type    = "pd-standard"
+
+## GKE Application Node Pool Settings
+application_node_pool_machine_type = "n2-highmem-4"
+application_node_pool_locations    = "us-central1-a"
+application_node_pool_min_count    = 1
+application_node_pool_max_count    = 1
```

Comment:
Minimum production viable infra here would be at least 3 nodes over 3 availability zone.